src/octopeer-github/main/ElementSelectionBehaviour/ButtonElementSelectionBehaviour/ButtonElementSelectionBehaviour.ts

```diff
@@ -1,17 +1,14 @@
-/// <reference path="ElementSelectionBehaviour.ts"/>
-/// <reference path="../DatabaseAdaptable/DatabaseAdaptable.ts"/>
-/// <reference path="../ElementEventBinding/ElementEventBinding.ts"/>
-
-/**
- * Created by Youri on 03/05/2016.
- */
-
+/// <reference path="../ElementSelectionBehaviour.ts"/>
+/// <reference path="../../DatabaseAdaptable/DatabaseAdaptable.ts"/>
+/// <reference path="../../ElementEventBinding/ElementEventBinding.ts"/>
 /**
+ * Created by Mathias on 2016-05-11.
  * Class for selecting buttons.
  */
-class ButtonsElementSelectionBehaviour implements ElementSelectionBehaviour {
-    private elementDsc: string = ".btn";
-    private elementID: ElementID = new ElementID(1);
+class ButtonElementSelectionBehaviour implements ElementSelectionBehaviour {
+
+    public elementDsc: string;
+    public elementID: ElementID;
```

Comment:
Unfortunately, TypeScript doesn't support the enforcement of getters and setters in interfaces, so this is a no-go :/